homeassistant/components/honeywell/climate.py

```diff
@@ -53,6 +53,13 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     if region == 'us':
         return _setup_us(username, password, config, add_entities)
 
+    _LOGGER.warning(
+        "The honeywell component is deprecated for EU (i.e. non-US) systems, "
+        "this functionality will be removed in a future version of HA.")
```

Comment:
Spell out "Home Assistant" (or just remove the last two words.)

Reply:
Done